<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="1" ignored="1" total="7" passed="4" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-03-26T11:37:54 IST" name="Suite" finished-at="2024-03-26T11:39:28 IST" duration-ms="94142">
    <groups>
      <group name="purchase">
        <method signature="E2ETest.SubmitOrder(java.util.HashMap)[pri:0, instance:alasana.tests.E2ETest@531f4093]" name="SubmitOrder" class="alasana.tests.E2ETest"/>
      </group> <!-- purchase -->
      <group name="errorHandling">
        <method signature="errorValidationTest.loginErrorValidation()[pri:0, instance:alasana.tests.errorValidationTest@5629510]" name="loginErrorValidation" class="alasana.tests.errorValidationTest"/>
      </group> <!-- errorHandling -->
    </groups>
    <test started-at="2024-03-26T11:37:54 IST" name="End To End Test" finished-at="2024-03-26T11:38:38 IST" duration-ms="44243">
      <class name="alasana.tests.E2ETest">
        <test-method is-config="true" signature="LaunchApplication()[pri:0, instance:alasana.tests.E2ETest@531f4093]" started-at="2024-03-26T11:37:54 IST" name="LaunchApplication" finished-at="2024-03-26T11:38:10 IST" duration-ms="16168" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LaunchApplication -->
        <test-method signature="SubmitOrder(java.util.HashMap)[pri:0, instance:alasana.tests.E2ETest@531f4093]" started-at="2024-03-26T11:38:10 IST" name="SubmitOrder" data-provider="getData" finished-at="2024-03-26T11:38:16 IST" duration-ms="5759" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Sana@123, email=alasana@gmail.com, productName=ZARA COAT 3}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SubmitOrder -->
        <test-method is-config="true" signature="Close()[pri:0, instance:alasana.tests.E2ETest@531f4093]" started-at="2024-03-26T11:38:16 IST" name="Close" finished-at="2024-03-26T11:38:16 IST" duration-ms="369" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Close -->
        <test-method is-config="true" signature="LaunchApplication()[pri:0, instance:alasana.tests.E2ETest@531f4093]" started-at="2024-03-26T11:38:16 IST" name="LaunchApplication" finished-at="2024-03-26T11:38:26 IST" duration-ms="9765" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LaunchApplication -->
        <test-method signature="SubmitOrder(java.util.HashMap)[pri:0, instance:alasana.tests.E2ETest@531f4093]" started-at="2024-03-26T11:38:26 IST" name="SubmitOrder" data-provider="getData" finished-at="2024-03-26T11:38:31 IST" duration-ms="4951" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Sana@123, email=sana301@gmail.com, productName=ADIDAS ORIGINAL}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SubmitOrder -->
        <test-method is-config="true" signature="Close()[pri:0, instance:alasana.tests.E2ETest@531f4093]" started-at="2024-03-26T11:38:31 IST" name="Close" finished-at="2024-03-26T11:38:32 IST" duration-ms="414" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Close -->
        <test-method is-config="true" signature="LaunchApplication()[pri:0, instance:alasana.tests.E2ETest@531f4093]" started-at="2024-03-26T11:38:32 IST" name="LaunchApplication" finished-at="2024-03-26T11:38:37 IST" duration-ms="5231" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LaunchApplication -->
        <test-method signature="orderHistory()[pri:0, instance:alasana.tests.E2ETest@531f4093]" started-at="2024-03-26T11:38:37 IST" name="orderHistory" finished-at="2024-03-26T11:38:38 IST" depends-on-methods="alasana.tests.E2ETest.SubmitOrder" duration-ms="1036" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- orderHistory -->
        <test-method is-config="true" signature="Close()[pri:0, instance:alasana.tests.E2ETest@531f4093]" started-at="2024-03-26T11:38:38 IST" name="Close" finished-at="2024-03-26T11:38:38 IST" duration-ms="248" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Close -->
      </class> <!-- alasana.tests.E2ETest -->
    </test> <!-- End To End Test -->
    <test started-at="2024-03-26T11:38:39 IST" name="error Validation" finished-at="2024-03-26T11:39:28 IST" duration-ms="49414">
      <class name="alasana.tests.errorValidationTest">
        <test-method is-config="true" signature="LaunchApplication()[pri:0, instance:alasana.tests.errorValidationTest@5629510]" started-at="2024-03-26T11:38:39 IST" name="LaunchApplication" finished-at="2024-03-26T11:38:46 IST" duration-ms="7265" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LaunchApplication -->
        <test-method retried="true" signature="loginErrorValidation()[pri:0, instance:alasana.tests.errorValidationTest@5629510]" started-at="2024-03-26T11:38:46 IST" name="loginErrorValidation" finished-at="2024-03-26T11:39:08 IST" duration-ms="22177" status="SKIP">
          <exception class="org.openqa.selenium.TimeoutException">
            <message>
              <![CDATA[Expected condition failed: waiting for visibility of Proxy element for: DefaultElementLocator 'By.cssSelector: [class*='flyInOut']' (tried for 5 second(s) with 500 milliseconds interval)]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for visibility of Proxy element for: DefaultElementLocator 'By.cssSelector: [class*='flyInOut']' (tried for 5 second(s) with 500 milliseconds interval)
at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:84)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:230)
at alasana.Abstractcomponents.AbstractComponents.waitForWebElementToAppear(AbstractComponents.java:50)
at alasana.pageobjects.LandingPage.geterrorMessage(LandingPage.java:45)
at alasana.tests.errorValidationTest.loginErrorValidation(errorValidationTest.java:22)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"css selector","selector":"[class*='flyInOut']"}
  (Session info: chrome=122.0.6261.131)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: '4.11.0', revision: '040bc5406b'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.7'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [ee7515a6b0a92f37813d150c4b821aa0, findElement {using=css selector, value=[class*='flyInOut']}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 122.0.6261.131, chrome: {chromedriverVersion: 122.0.6261.128 (f18a44fedeb..., userDataDir: C:\Users\alasa\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:51672}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:51672/devtoo..., se:cdpVersion: 122.0.6261.131, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: ee7515a6b0a92f37813d150c4b821aa0
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:52)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:191)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:196)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:171)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:518)
at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:165)
at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:59)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:355)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:349)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:68)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at jdk.proxy2/jdk.proxy2.$Proxy19.isDisplayed(Unknown Source)
at org.openqa.selenium.support.ui.ExpectedConditions.elementIfVisible(ExpectedConditions.java:304)
at org.openqa.selenium.support.ui.ExpectedConditions.access$000(ExpectedConditions.java:37)
at org.openqa.selenium.support.ui.ExpectedConditions$10.apply(ExpectedConditions.java:290)
at org.openqa.selenium.support.ui.ExpectedConditions$10.apply(ExpectedConditions.java:287)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:205)
... 31 more
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.TimeoutException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginErrorValidation -->
        <test-method is-config="true" signature="Close()[pri:0, instance:alasana.tests.errorValidationTest@5629510]" started-at="2024-03-26T11:39:08 IST" name="Close" finished-at="2024-03-26T11:39:08 IST" duration-ms="169" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Close -->
        <test-method is-config="true" signature="LaunchApplication()[pri:0, instance:alasana.tests.errorValidationTest@5629510]" started-at="2024-03-26T11:39:08 IST" name="LaunchApplication" finished-at="2024-03-26T11:39:14 IST" duration-ms="5755" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LaunchApplication -->
        <test-method signature="loginErrorValidation()[pri:0, instance:alasana.tests.errorValidationTest@5629510]" started-at="2024-03-26T11:39:14 IST" name="loginErrorValidation" finished-at="2024-03-26T11:39:15 IST" duration-ms="1100" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or password.] but found [Incorrect email password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or password.] but found [Incorrect email password.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at alasana.tests.errorValidationTest.loginErrorValidation(errorValidationTest.java:22)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:248)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginErrorValidation -->
        <test-method is-config="true" signature="Close()[pri:0, instance:alasana.tests.errorValidationTest@5629510]" started-at="2024-03-26T11:39:16 IST" name="Close" finished-at="2024-03-26T11:39:16 IST" duration-ms="180" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Close -->
        <test-method is-config="true" signature="LaunchApplication()[pri:0, instance:alasana.tests.errorValidationTest@5629510]" started-at="2024-03-26T11:39:16 IST" name="LaunchApplication" finished-at="2024-03-26T11:39:24 IST" duration-ms="8022" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LaunchApplication -->
        <test-method signature="productErrorValidation()[pri:0, instance:alasana.tests.errorValidationTest@5629510]" started-at="2024-03-26T11:39:24 IST" name="productErrorValidation" finished-at="2024-03-26T11:39:28 IST" duration-ms="3930" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- productErrorValidation -->
        <test-method is-config="true" signature="Close()[pri:0, instance:alasana.tests.errorValidationTest@5629510]" started-at="2024-03-26T11:39:28 IST" name="Close" finished-at="2024-03-26T11:39:28 IST" duration-ms="290" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Close -->
      </class> <!-- alasana.tests.errorValidationTest -->
    </test> <!-- error Validation -->
  </suite> <!-- Suite -->
</testng-results>
